### ISSUE TITLE
Host DevTools on own permalink page via <iframe>

### ISSUE THREAD

**Issue by auchenberg** [auchenberg/devtools-remote#5]
- Build session permalink page with some branding
- Load Hosted DevTools via <iframe> from AppSpot

**Comment by kzahel** [comment]
I'd love to be able to load specific devtools revisions (or any at all) in a http(s) URL. I tried using https://github.com/ChromeDevTools/devtools-frontend but the package.json did not have any scripts that seemed relevant to building the static html/css/js assets to be able to host a devtools instance. Any tips?

**Comment by auchenberg** [comment]
Chrome is providing a cloud hosted front-end on AppSpot, so that would be ideal. Wanna try to give it a spin @kzahel ?

**Comment by kzahel** [comment]
Definitely! I saw that it was using appcache to download/store the assets. Would I just have it do that again and copy the URL format?

**Comment by kzahel** [comment]
Hmm I found a URL that seems to work (https://chrome-devtools-frontend.appspot.com/serve_file/@d19c087b9bfdf76a8af91e2f0a0e791bf2ffcafa/inspector.html)

But it seems it doesn't include emulated mobile devices module?

`Application Cache Error event: Resource fetch failed (404) https://chrome-devtools-frontend.appspot.com/serve_file/@d19c087b9bfdf76a8af91e2f0a0e791bf2ffcafa/emulated_devices/emulated_devices_module.js`

**Comment by kzahel** [comment]
I tried running `scripts/compile_frontend.py` but it wanted v8 and other things checked out so I compiled Chrome. Now I have a compiled frontend in my build output folder. However there is no AppCache manifest created. Is the script mentioned at https://groups.google.com/a/chromium.org/d/msg/devtools-reviews/3izWlIsoIl8/v-rhJ6upkNYJ (one that injects manifest url into inspector.html) somewhere in the chromium repository? or is that somewhere else?
